protocol/x/bridge/keeper/grpc_query_test.go

```diff
@@ -254,7 +254,7 @@ func TestDelayedCompleteBridgeMessages(t *testing.T) {
 
 			// Construct expected responses.
 			delayMsgAuthority := authtypes.NewModuleAddress(delaymsgtypes.ModuleName).String()
-			blockOfExecution := int64(k.GetSafetyParams(ctx).DelayBlocks) + ctx.BlockHeight()
+			blockOfExecution := k.GetSafetyParams(ctx).DelayBlocks + uint32(ctx.BlockHeight())
```

Comment:
nit: lib.MustConvertIntegerToUint32? 

not a big deal though

Reply:
Let me fix this consistently in the codebase in a new PR